.travis.yml

```diff
@@ -57,9 +57,12 @@ before_install:
     export BOTO_CONFIG=/dev/null;
     openssl aes-256-cbc -K $encrypted_29f3b7c4d8c3_key -iv $encrypted_29f3b7c4d8c3_iv  -in resources_for_CI/servicekey.json.enc -out servicekey.json -d;
     scripts/install_gcloud.sh;
-    printf 'y\n' | $GCLOUD/gcloud components update app;
-    $GCLOUD/gcloud config set project broad-dsde-dev;
-    $GCLOUD/gcloud auth activate-service-account --key-file servicekey.json;
+    printf 'y\n' | $GCLOUD/gcloud components update;
+    if [[ $TEST_TYPE == cloud ]]; then
```

Comment:
Could you have a look at the existing tests marked as "cloud" to make sure the label is being applied consistently?